crates/egui-wgpu/src/egui.wgsl

```diff
@@ -87,5 +114,6 @@ fn fs_main_gamma_framebuffer(in: VertexOutput) -> @location(0) vec4<f32> {
     let tex_linear = textureSample(r_tex_color, r_tex_sampler, in.tex_coord);
     let tex_gamma = gamma_from_linear_rgba(tex_linear);
     let out_color_gamma = in.color * tex_gamma;
-    return out_color_gamma;
+    let out_color_dithered = vec4<f32>(dither_interleaved(out_color_gamma.rgb, 256.0, in.position * r_locals.pixels_per_point), out_color_gamma.a);
```

Comment:
I don't get what `* r_locals.pixels_per_point` is about here. `in.position` is `@builtin(position)` which makes this raw pixel coordinates ranging from 0 to <target size>, see https://www.w3.org/TR/WGSL/#built-in-values-position. So scaling it with eguis points->pixel factor doesn't make any sense since we're starting with pixels anyways.

you also didn't apply this to either glsl or the `fs_main_linear_framebuffer` version. What's the intention here?